cmd/policy/pull.go

```diff
@@ -4,13 +4,15 @@ import "github.com/pkg/errors"
 
 type PullCmd struct {
 	Policies []string `arg:"" name:"policy" help:"Policies to pull from the remote registry."`
+	Untar    bool     `name:"untar" help:"Untar the policy bundle after pulling."`
+	UntarDir string   `name:"untardir" help:"Directory to extract the policy bundle." default:"." type:"path"`
```

Comment:
From a security POV, I would require that `--untardir` must point to an existing directory. 

To enforce this, please use `type:"existingdir"` instead of `type:"path"` to leverage the existence check present in kong package.